### DISCUSSION TITLE
Serial BMS connection

### DISCUSSION THREAD

**Discussion by Bohne412** [mr-manuel/venus-os_dbus-serialbattery#83] [General]
Hello

I can no longer get my 3 battery to work. Serial BMS connection there is a page on https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/troubleshoot/ but unfortunately I can't get it to work.

Maybe someone here has an idea.

Warm regards

**Comment by mr-manuel** [comment]
Verify your connection cable

**Reply by Bohne412** [reply]
They are fine. Unfortunately, I can't understand the instructions. Cables are fine .

**Reply by mr-manuel** [reply]
Why can't you understand it? Because of the language or tecnical skills?
